### PR TITLE
perf: added error handling missing code 

### PR DESCRIPTION
**Pull request by Pranshu-Raj**
### ⚠️ &nbsp;&nbsp;Pre Checklist

> Please complete _ALL_ items in this checklist, and remove before submitting

- [x] I have read through the [Contributing](https://devlake.apache.org/community/) Documentation & [PR Template](https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue)
- [x] This PR is using a `label` (bug, feature etc.)
- [x] My code is has necessary documentation (if appropriate)
- [x] I have added any relevant tests
- [x] This section (**⚠️ &nbsp;&nbsp;Pre Checklist**) will be removed when submitting PR

# Summary
Added error handling missing code in gitlab/api/connections.go (GET /plugins/gitlab/connections/:connectionId)
Added error handling missing code in gitlab/jenkins/connections.go (GET /plugins/jenkins/connections/:connectionId)
Added error handling missing code in gitlab/jira/connections.go (GET /plugins/jira/connections/:connectionId)
Added error handling missing code in gitlab/tapd/connections.go (GET /plugins/tapd/connections/:connectionId)

### Does this close any open issues?
[Refactor][gitlab,jira,jenkins,tapd] Missing error handling #2561

### Screenshots
Include any relevant screenshots here.

### Other Information
Any other information that is important to this PR.

### CODE REVIEW

**Comment by mappjzc** [comment]
image
Whether error is empty or not, error is returned.
So we don't need to make "err ! = nil" judgment before this.

File: plugins/tapd/api/connection.go [hunk 1]
```diff
@@ -161,5 +161,8 @@ GET /plugins/tapd/connections/:connectionId
 func GetConnection(input *core.ApiResourceInput) (*core.ApiResourceOutput, error) {
 	connection := &models.TapdConnection{}
 	err := connectionHelper.First(connection, input.Params)
-	return &core.ApiResourceOutput{Body: connection}, err
+	if err != nil {
+		return nil,err
+	}
+	return &core.ApiResourceOutput{Body: connection},nil
```
Comment (abeizn) [inline plugins/tapd/api/connection.go:167]: please keep in line with other plugins, and missing space.

Comment (Pranshu-Raj) [reply plugins/tapd/api/connection.go:167]: okay that i did previously but why this lint comments test is failing can u help me with that?

Comment (abeizn) [reply plugins/tapd/api/connection.go:167]: both 165 and 167 need to add space, you added the space at 167 and missing 165, it maybe lint commit-msg test is failing.

Comment (keon94) [reply plugins/tapd/api/connection.go:167]: try running `make fmt`. the go formatter should take care of these kind of things

Comment (klesh) [reply plugins/tapd/api/connection.go:167]: seems like this is the only issue left for this PR, do you plan to work on it? @Pranshu-Raj

Comment (Pranshu-Raj) [reply plugins/tapd/api/connection.go:167]: sorry @klesh I am unable to resolve this issue.

Comment (klesh) [reply plugins/tapd/api/connection.go:167]: No problem. I will try to fix the rest

**Review by abeizn — APPROVED** [review]
LGTM

**Pull request merged**